tensorboard/webapp/routes/dashboard_deeplink_provider.ts

```diff
@@ -42,6 +42,7 @@ import {
 import {featureFlagsToSerializableQueryParams} from './feature_flag_serializer';
 
 const COLOR_GROUP_REGEX_VALUE_PREFIX = 'regex:';
+const COLOR_GROUP_REGEX_EXP_VALUE_PREFIX = 'regex_exp:';
```

Comment:
nit: regex_exp can very easily be misread as "regex expression", which can cause unnecessary confusion. Suggest the full: `COLOR_GROUP_REGEX_BY_EXP_VALUE_PREFIX = 'regex_by_exp:'`

Reply:
Oh, that's a good catch. Thank you!